atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -142,7 +142,7 @@ public String getKeyspaceOrThrow() {
     @Deprecated
     public abstract Optional<String> keyspace();
 
-    public abstract Optional<CassandraCredentialsConfig> credentials();
+    CassandraCredentialsConfig credentials();
```

Comment:
ah I guess we moved this from an abstract class to an interface on develop. will fix